druid-shell/src/keyboard.rs

```diff
@@ -37,6 +37,37 @@ pub struct KeyEvent {
     //TODO: add time
 }
 
+/// A modifier, for the purpose of matching key events.
+pub enum Modifier {
```

Comment:
This has the same issue, I think, that is addressed by the `SysMods` and `RawMods` types in `hotkey.rs`: if we want to match 'undo' we need some key that is 'cmd' on macOS and 'ctrl' elsewhere.